sql/updates/world/master/9999_99_99_99_world .sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `spell_area` CHANGE COLUMN `autocast` `flags` TINYINT(3) UNSIGNED NOT NULL DEFAULT '3' AFTER `gender`;
+UPDATE `spell_area` SET `flags` = `flags` + 2;
```

Comment:
```sql
UPDATE `spell_area` SET `flags` = `flags` & 2;
```

Reply:
flags | 2, its intended here that they are added

Reply:
yeah, thats what i meant ^^

Reply:
Ya ok :D.